src/test/java/de/retest/recheck/review/ignore/AttributeFilterLoaderTest.java

```diff
@@ -0,0 +1,34 @@
+package de.retest.recheck.review.ignore;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import de.retest.recheck.ignore.AllMatchFilter;
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.review.ignore.AttributeFilter.AttributeFilterLoader;
+import de.retest.recheck.review.ignore.AttributeFilter.LegacyAttributeFilterLoader;
+
+class AttributeFilterLoaderTest extends LegacyAttributeFilterLoader {
+
+	final AttributeFilterLoader cut = new AttributeFilterLoader();
+
+	@Test
+	void save_should_produce_same_as_was_given_to_load_for_simple_line() {
+		final String line = "attribute=text";
+		assertThat( cut.load( line ).map( filter -> cut.save( filter ) ).get() ).isEqualTo( line );
+	}
+
+	@Test
+	void save_should_produce_same_as_was_given_to_load_for_chained_filter() {
+		final String line = "attribute=outline, pixel-diff=5px";
+		assertThat( cut.load( line ).map( filter -> cut.save( filter ) ).get() ).isEqualTo( line );
+	}
+
+	@Test
+	void chained_filters_should_load_correctly() {
+		final Filter loaded = cut.load( "attribute=outline, pixel-diff=5px" ).get();
```

Comment:
More idiomatic in terms of AssertJ:

```java
assertThat( cut.load( "attribute=outline, pixel-diff=5px" ) ).hasValueSatisfying( loaded -> {
	assertThat( loaded ).isInstanceOf( AllMatchFilter.class );
	assertThat( ((AllMatchFilter) loaded).getFilters() ).hasSize( 2 );
} );

Same elsewhere.

Reply:
I think that's less readable. Golden Hammer Syndrome :-D